### DISCUSSION TITLE
How to set serial-port to fixed value?

### DISCUSSION THREAD

**Discussion by sven-probst** [Kannix2005/homeassistant-selve#24] [Q&A]
Hello,

is it possible to set the serial-port in homeassistant-config to a fixed value, because actual I think it is probing the devices. This is maybe a problem in my environment.

Best regards

Sven

**Comment by frankfrommelt** [comment]
Well, I was lucky as "in the old times" (when I started with this integration) you could (and had to) manually configure to device as a linux path (e.g. /dev/...).
I was a bit astonished this option is no longer available.
Maybe you have a chance to start with an older version of the integration that allowed a manual configuration.
I have to admit I have no idea where the integrations store their configurations. I guess somewhere in the database...
